.gitmodules

```diff
@@ -140,3 +140,6 @@
 [submodule "tools/rocket-dsp-utils"]
 	path = tools/rocket-dsp-utils
 	url = https://github.com/ucb-bar/rocket-dsp-utils
+[submodule "generators/ibex"]
+	path = generators/ibex
+	url = https://github.com/schwarz-em/ibex-wrapper
```

Comment:
This wrapper repo should be added to `ucb-bar`. Then you should change this link.

Reply:
Also can you open that new repo with a PR with all the new code so that I can comment/review that code.